docs/05-access-management/index.mdx

```diff
@@ -4,12 +4,48 @@ import WipWarning from '../_partials/_wip.mdx';
 
 # Access Management
 
-<WipWarning>
-  <p>The current version of the access management dashboard is only provided to support our standalone <a href="/docs/encryption/">encryption libraries</a>.</p>
-  <p>A new dashboard allowing you to manage all aspects of blindnet devkit applications and users, including encryption, will be soon available.</p>
-  <p>In the mean time, we provide a <a href="https://devkit-pce-staging.azurewebsites.net/swagger/">pre-configured Privacy Computation Engine</a> to be used directly or via other components of the DevKit without any need for an application id. Check out our <a href="../tutorial">Quick Start Tutorial</a> for more details.</p>
+## Set Up your Privacy System
+
+<WipWarning callToAction="false">
+  <p>The instruction in this section are only here to give you a glips on how the system identification works with the Privacy Computation Engine.</p>
+  <p>At this moment, one blindnet devkit install using a single instance of the Privacy Computation Engine <em>is</em> considered as a coherent single system automatically, using an internally generated ID.</p>
+  <p>Check out out <a href="/docs/tutorial">introductory tutorial</a> for a concrete and up-to-date overview.</p>
 </WipWarning>
 
----
+In blindnet devkit, all components are identified to be part of a coherent "system" using a "System ID".
```

Comment:
Mention that components belonging to a single system id easily communicate, while it's hard for components of different systems (only through transfers and privacy requests having `target` specified). So it's the best to have one devkit system id for one logical system (that encapsulates the data).